composer.json

```diff
@@ -17,8 +26,8 @@
   "minimum-stability": "dev",
   "prefer-stable": true,
   "scripts": {
-    "cs-check": "phpcs -p -s --standard=vendor/spryker/code-sniffer/SprykerStrict/ruleset.xml src/ tests/",
-    "cs-fix": "phpcbf -p --standard=vendor/spryker/code-sniffer/SprykerStrict/ruleset.xml src/ tests/"
+    "cs-check": "phpcs -p -s --standard=vendor/spryker/code-sniffer/SprykerStrict/ruleset.xml src/",
```

Comment:
this is not valid, why removing tests?

Reply:
whats the issue here?

Reply:
there are no tests at the moment. Therefore, the validation is failed
https://travis-ci.com/github/spryker/suite-nonsplit/jobs/366104567

 if [[ $VALIDATION == 1 ]] ; then vendor/bin/console dev:composer:update-json-files -d -v ; fi
Store: DE | Environment: devtest
1 of 855 module(s) need(s) updating.
- SprykerSdk.Benchmark
Please run this command locally without dry-run:
vendor/bin/console dev:composer:update-json-files SprykerSdk.Benchmark
The command "if [[ $VALIDATION == 1 ]] ; then vendor/bin/console dev:composer:update-json-files -d -v ; fi" exited with 1.

Reply:
I assumed you have some smoke tests..
Ok then.